src/platform/test/functional/apps/discover/tabs/_save_and_load.ts

```diff
@@ -149,6 +151,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await browser.refresh();
         await discover.waitUntilTabIsLoaded();
 
+        await discover.setChartInterval(nonAutoChartIntervalTitle);
+        const selectedInterval = await discover.getChartInterval();
+        expect(selectedInterval).to.not.be(null);
+        persistedTabChartIntervalValue = selectedInterval!;
+        expect(persistedTabChartIntervalValue).to.not.be('auto');
```

Comment:
```suggestion
        await discover.setChartInterval(persistedTabChartIntervalTitle);
```
Could we move this to the `// Persisted data view tab` section for consistency with the rest of the test, and to ensure it's restored after a page refresh like the other properties? I also don't think we need the additional lines if we use a hardcoded value since we already validate it below.